django/views/debug.py

```diff
@@ -101,6 +101,9 @@ class ExceptionReporterFilter:
     contain lenient default behaviors.
     """
 
+    def get_safe_request_meta(self, request):
```

Comment:
Please test this method (coverage report indicates that the other methods in this class aren't tested so don't look for existing similar tests.